### ISSUE TITLE
Accessibility: After opening a dialog with a close button, focus should be on the content, not the close button

### ISSUE THREAD

**Issue by rickyp-ms** [OfficeDev/office-ui-fabric-react#2544]
### Bug Report

- __Package version(s)__: 4.21.3
- __Browser and OS versions__: All

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: tasks.office.com

#### Describe the issue:

Accessibility request: When opening a dialog, the first focus should be on the content, not the close button

#### Actual behavior:

Focus is on the close button

#### Expected behavior:

Focus to be on something useful, like information in the dialog

### If applicable, please provide a codepen repro:

It's been this way for a while, even on the old site. http://dev.office.com/fabric#/components/dialog . See the first example (tabbing to that button helps you see the focus better),
